backend/dataall/modules/notebooks/services/notebook_service.py

```diff
@@ -42,10 +42,10 @@
 class NotebookCreationRequest:
     """A request dataclass for notebook creation. Adds default values for missed parameters"""
 
-    label: str
-    VpcId: str
-    SubnetId: str
-    SamlAdminGroupName: str
+    label: str = None
+    VpcId: str = None
+    SubnetId: str = None
+    SamlAdminGroupName: str = None
```

Comment:
why these changes to default `None`?

Reply:
Yes, let me revert that and add these exception to the edge cases of PR part2